projects/range/index.js

```diff
@@ -4,12 +4,14 @@ const ABI = require('./abi.json');
 const config ={
   ethereum: [
     { factory: '0xf1e70677fb1f49471604c012e8B42BA11226336b', fromBlock: 17266660 }, // uniswap
-    { factory: '0x3edeA0E6E94F75F86c62E1170a66f4e3bD7d77fE', fromBlock: 18460401 } // pancakeswap
+    { factory: '0x3edeA0E6E94F75F86c62E1170a66f4e3bD7d77fE', fromBlock: 18460401 }, // pancakeswap
+    { factory: '0xDE07a0D5C9CA371E41a869451141AcE84BCAd119', fromBlock: 18375548 }, // GHO
```

Comment:
so GHO vaults dont have `getUnderlyingBalances` but `getUnderlyingBalancesFromPool` ?

Reply:
you're right. Multicall is failing too because factory contract for pancakeswap is the same as Izumi, which is calling the wrong ABI. Refactoring the logic to support these cases 🙇